### ISSUE TITLE
Your shop is being processed

### ISSUE THREAD

**Issue by opanga328** [facebookincubator/facebook-for-woocommerce#1060]
Hi,

I am experiencing the issue "Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."

Kindly look into this for me.

https://www.facebook.com/dmdfashionCA/
https://www.facebook.com/pg/dmdfashionCA/shop/?ref=page_internal
https://dmdfashion.ca/shop/

**Comment by Luther2010** [comment]
## Automated Support Message

Hello,

Thanks for your interest in our plugin! Our developer team tries to provide personal support on GitHub whenever possible.

If you have a feature request or feedback for the developers, reply with `Feature Request` or `Feedback`.
If you are a developer or have a question for the developers related to code, reply with `Developer Question`.
If you are running ads with us, reply with `Advertiser`.

If you have encountered a bug or a problem with the plugin, keep reading for information that can help you resolve the issue yourself or provide us details we need to help you.

## Search Existing Issues

First, please [search our issues](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) to see if your problem has been reported before.
There may already be a workaround or a solution to your problem. You can use the [search bar](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) near the top of our issues page. Be sure to search both _open_ and _closed_ issues.
If you do find a similar issue, leave a comment on that issue instead of creating a new one. Your comment helps us prioritize which existing issues are important and urgent.

## Common Resolution Methods

If you are experiencing a problem or bug and have not found any similar issue, please try the following:

* Upgrade to the latest version of the plugin
* Try to `Resync` products using the button on the plugin config page
* Reset the plugin by selecting `Manage Settings > Advanced Options > Delete` from the configuration page of the plugin and setting up again
* Try deactivating other plugins that you think may be interfering, then reset and set up again

## Necessary Information

If the above steps did not help, please provide the following information in your next comment in order for us to debug your problem:

* WooCommerce version
* WordPress version
* Plugin version (please note: we only support up-to-date versions of the plugin)
* A screenshot of where you see the problem (for example: if it is on Facebook, provide a screenshot of Facebook, if it is on WooCommerce, provide a screenshot of WooCommerce. You can provide multiple screenshots if the problem relates to different screens.)
* The URL of your website
* The URL of your Facebook Page
* The URL(s) of where you took your screenshot(s)
* If applicable, detailed steps you took in order to see the problem

Please include `Debug Ready` in your reply so our automated system can note your issue for manual review. Please be aware that due to a high volume of issues, we may not be able to respond immediately.

## Response Time

We prioritize issues with the tags `Advertiser` or `Multiple Reports`.
For issues that have the tags `Debug Ready`, `Feature Request` or `Developer Question`, we aim to respond within 30 days.

If you have not responded to our automated message with one of the above phrases, such as `Debug Ready`, we may close your issue after 30 days. If a `Feature Request` is inactive (without new requests, messages or pull requests) for 6 months, we may close it.

Please note that we are not always able to provide support for issues related to interaction from other plugins with this plugin or related to the Facebook advertising interface. To get support related to Facebook ads, try visiting the [Facebook Ads Help Center](https://www.facebook.com/business/help).

For more information on our response times, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by opanga328** [comment]
Multiple Reports

**Comment by opanga328** [comment]
WooCommerce version : Version 3.6.4
WordPress version : 5.2.2
Plugin version (please note: we only support up-to-date versions of the plugin) : 1.9.15 
A screenshot of where you see the problem (for example: if it is on Facebook, provide a screenshot of Facebook, if it is on WooCommerce, provide a screenshot of WooCommerce. You can provide multiple screenshots if the problem relates to different screens.)

![image](https://user-images.githubusercontent.com/52262307/60306899-3ccb6580-9910-11e9-8688-1346a15d9577.png)

![image](https://user-images.githubusercontent.com/52262307/60306963-8451f180-9910-11e9-8263-038854b52b00.png)

![image](https://user-images.githubusercontent.com/52262307/60307000-a0559300-9910-11e9-8cec-dbabebb058e8.png)

The URL of your website : https://dmdfashion.ca/shop/
The URL of your Facebook Page : https://www.facebook.com/dmdfashionCA/
The URL(s) of where you took your screenshot(s) : https://www.facebook.com/pg/dmdfashionCA/shop/?ref=page_internal

If applicable, detailed steps you took in order to see the problem

**Comment by opanga328** [comment]
I set this up 3 days ago. Its been more than 48 hours. I appreciate your help.

**Comment by opanga328** [comment]
Hi, 

Is there any update on this issue?

Thanks.

**Comment by opanga328** [comment]
Advertiser

**Comment by Tanvir294** [comment]
Hi,

Same Problem.......

I am experiencing the issue "Your shop is being processed
Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."

Kindly look into this for me.

Website: https://houseofheiress.store/shop-2/
Facebook Page: https://business.facebook.com/houseofheiresscouture/
https://business.facebook.com/pg/houseofheiresscouture/shop/

WooCommerce version : Version 3.6.4
Facebook for WooCommerce: Version 1.9.15
WordPress version : 5.2.2
![House of Heiress](https://user-images.githubusercontent.com/52356940/60387693-cde53c80-9ac8-11e9-8a41-c2e26b8bfb64.jpg)

**Comment by hwtmusic** [comment]
Hi,

I have the same problem.  I have submitted 3 support tickets from facebook with zero response.  It's been in the "Your shop is being processed" for over 2 weeks.

What I can see: https://www.facebook.com/pg/hwtmusic/shop/?ref=page_internal
Users can see: https://www.facebook.com/hwtmusic
![20190702_205758](https://user-images.githubusercontent.com/52093803/60555841-923ab480-9d0c-11e9-990f-a8777fa0f80f.jpg)

Actual shop using woocommerce: http://hwtmusic.com/shop

**Comment by DeannaAnuar** [comment]
I am facing the same issue too.

I've the latest **Facebook for Woocommerce version 1.9.15**, **Woocommerce v 3.6.5** and synced my products on 30th June. Until today my Facebook Shop is still processing.
![Pending Approval](https://user-images.githubusercontent.com/52489428/60579526-dc2ba500-9db5-11e9-80d6-19e81875aa7f.JPG)


![WC 2](https://user-images.githubusercontent.com/52489428/60579646-1d23b980-9db6-11e9-8272-c64f027c420a.JPG)
![Capture](https://user-images.githubusercontent.com/52489428/60579701-3cbae200-9db6-11e9-9b73-9e9d8bc49712.JPG)

I've emailed Facebook and was told to wait 48 hours. But up till today my Shop is still not synced.

Website: https://brushnco.com/
Facebook Page: https://www.facebook.com/brushnco/
Facebook Shop link: https://www.facebook.com/pg/brushnco/shop/?ref=page_internal

Thank you!

**Comment by danlowp** [comment]
Same problem here:
Website: https://lowpressure.co.uk
Facebook Page: https://www.facebook.com/pg/stormridersurf/
Facebook for WooCommerce: Version 1.9.15
WooCommerce: Version 3.6.4
![Screen Shot ](https://user-images.githubusercontent.com/52530684/60658836-f021e600-9e4b-11e9-8161-c4029f79d4b8.png)

Please help, thanks.

**Comment by hwtmusic** [comment]
> Hi,
> 
> I am experiencing the issue "Your shop is being processed
> Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."
> 
> Kindly look into this for me.
> 
> https://www.facebook.com/dmdfashionCA/
> https://www.facebook.com/pg/dmdfashionCA/shop/?ref=page_internal
> https://dmdfashion.ca/shop/

@opanga328 did you ever get a solution yet?

**Comment by hwtmusic** [comment]
Hello, is anyone out there??

**Comment by opanga328** [comment]
> Hi,
> I am experiencing the issue "Your shop is being processed
> Once processing is complete, your shop will be visible to people who visit your Page. You can still create and run ads promoting your products on Facebook."
> Kindly look into this for me.
> https://www.facebook.com/dmdfashionCA/
> https://www.facebook.com/pg/dmdfashionCA/shop/?ref=page_internal
> https://dmdfashion.ca/shop/
> 
> @opanga328 did you ever get a solution yet?

No reply at all. Not sure what else to do. 
Did anyone resolve this issue?

**Comment by opanga328** [comment]
@hwtmusic No reply at all. I still have the same issue and am not sure what else to do. 

Did anyone resolve this issue?

**Comment by opanga328** [comment]
@Luther2010 Please give us an update on this issue. All the relevant information has been provided. You help is appreciated.

**Comment by opanga328** [comment]
Hi @yinghec any update on this?

**Comment by opanga328** [comment]
@DeannaAnuar did you resolve this issue? If so, how did you do it? Please help.

**Comment by lgfogle** [comment]
Hey there! 👋

Lindsey here, from the SkyVerge product team! We recently took over development and support for the Facebook for WooCommerce extension.

In our recent 1.10 releases, we've fixed many issues impacting the store connection, pixel tracking, and product sync. Could you please update to the latest version of Facebook for WooCommerce and see if that resolves this issue?

If not, could you please [contact our support team](https://woocommerce.com/my-account/create-a-ticket/) or open a new issue here? We'd be happy to help troubleshoot further with you.

Thank you!
